frontend/src/scenes/notebooks/Nodes/NotebookNodeSurvey.tsx

```diff
@@ -0,0 +1,142 @@
+                            <StatusTag survey={survey as Survey} />
+                        </>
+                    )}
+                </div>
+
+                {expanded ? (
+                    <>
+                        {survey.description && (
+                            <>
+                                <LemonDivider className="my-0" />
+                                <span className="p-2">{survey.description}</span>
+                            </>
+                        )}
+                        {!survey.start_date ? (
+                            <>
+                                <LemonDivider className="my-0" />
+                                <div className="p-2">
+                                    <SurveyReleaseSummary id={id} survey={survey} hasTargetingFlag={hasTargetingFlag} />
+
+                                    <div className="w-full flex flex-col items-center">
+                                        <SurveyAppearance
+                                            type={survey.questions[0].type}
+                                            surveyQuestionItem={survey.questions[0]}
+                                            appearance={survey.appearance || defaultSurveyAppearance}
+                                            question={survey.questions[0].question}
+                                            description={survey.questions[0].description}
+                                            link={
+                                                survey.questions[0].type === SurveyQuestionType.Link
+                                                    ? survey.questions[0].link
+                                                    : undefined
+                                            }
+                                            readOnly={true}
+                                            onAppearanceChange={() => {}}
+                                        />
+                                    </div>
+                                </div>
+                            </>
+                        ) : (
+                            <>
+                                {/* show results when the survey is running */}
+                                <LemonDivider className="my-0" />
+                                <div className="p-2">
+                                    <SurveyResult disableEventsTable />
+                                </div>
+                            </>
+                        )}
+                    </>
+                ) : null}
+
+                <LemonDivider className="my-0" />
+                <div className="p-2 mr-1 flex justify-end gap-2">
+                    {survey.linked_flag && (
+                        <LemonButton
+                            type="secondary"
+                            size="small"
+                            icon={<IconFlag />}
+                            onClick={(e) => {
+                                e.stopPropagation()
```

Comment:
Why was this needed again? Ideally we should come up with a better way that doesn't force you to have remember to do this

Reply:
So when I click on this action, it doesn't expand the node as well, which also happens on click anywhere inside the component.